terminology/glossary.md

```diff
@@ -1,34 +1,35 @@
+|CSIRT| The Cmputer Security Incident Response Team is a group that provides organizations with services and support surrounding the assessment, management and prevention of cybersecurity-related emergencies|[NIST](https://csrc.nist.gov/glossary/term/computer_security_incident_response_team)|||
+|CNA|CVE Naming Authorities are groups authorized by the CVE Program to assign CVE IDs to vulnerabilities and publish CVE Records within their own specific scopes of coverage|[NIST](https://csrc.nist.gov/glossary/term/cna)|[CVE.org](https://www.cve.org/ProgramOrganization/CNAs)|
+|CVD|A Coordinated Vulenrability Disclosure coordinates the remediation and public disclosure of newly identified cybersecurity vulnerabilities||||
+|CVE|Common Vulnerabilities and Exposures is a dictionary of common names for publicly known information system vulnerabilities|[NIST](https://csrc.nist.gov/glossary/term/cve)|||
+|CVSS|Common Vulnerability Scoring System is a framework for communicating the characteristics and severity of software vulnerabilities|[NIST](https://csrc.nist.gov/glossary/term/common_vulnerability_scoring_system)|||
+|CWE|Common Weakness Enumeration is a category system for hardware and software weaknesses and vulnerabilities|[NIST](https://csrc.nist.gov/glossary/term/common_weakness_enumeration)|||
+|Disclosure policy| Provides guidelines for finders on conducting and submitting information on security vulnerabilities||||
+|Finder|||||
+|GHSA ID|||||
+|GSD|||||
+|Legal safe harbor|||||
+|Maintainer|||||
+|MITRE|An American not-for-profit organization supporting various U.S. government agencies in cybersecurity||||
+|MPCVD| A Multi-Party CVD is a coordinated vulenrability disclosure which spans multiple receiving organizations|[NTIA](https://www.ntia.doc.gov/other-publication/2016/multistakeholder-process-cybersecurity-vulnerabilities)|||
+|NDA|A Non-disclosure Agreement is a contract that prohibits the sharing of confidential information||||
+|NIST|The National Institute of Standards and Technology is a U.S. agency that promotes and maintains measurement standards||||
+|NVD|The National Vulnerability Database is a U.S. Government repository of standards-based vulnerability management data|[NIST](https://csrc.nist.gov/glossary/term/nvd)|||
+|OSS| Open Source Software is software with source code that anyone can inspect, modify, and enhance||||
+|OSV ID|||||
+|OpenSSF|The Open Source Security Foundation is a Linux Foundation, cross-industry forum collaborating to improve open source software security|[OSSF](https://openssf.org/about/faq/#)|||
+|POC|||||
+|PSIRT|A Product Security Incident Response Team is responsible for identifying, assessing and remediating risks associated with security vulnerabilities within the products of an organization|[NIST](https://csrc.nist.gov/glossary/term/product_security_incident_response_team)|||
+|Security policy|||||
```

Comment:
This definition is pretty weak, but it's something: https://csrc.nist.gov/glossary/term/security_policy.

"A set of criteria for the provision of security services."

Reply:
added